### PR TITLE
data: zakariyahrobiat

### PR DESCRIPTION
**Pull request by zakariyahrobiat**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3624"></a>

### CODE REVIEW

File: data/zakariyahrobiat.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+     {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/zakariyahrobiah",
+      "icon": "FaGithub"
+    },
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/m/in/robiat-zakariyah-743452225",
+      "icon": "FaLinkedIn"
```
Comment (senali-d) [inline data/zakariyahrobiat.json:21]: Linkedin icon name should be `FaLinkedin`

```suggestion
      "icon": "FaLinkedin"
```

**Review by senali-d — CHANGES_REQUESTED** [review]

File: data/zakariyahrobiat.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+      {
+      "name": "Follow me on instagram",
+      "url": "https://www.instagram.com/zakariyahrobiat2018",
+      "icon": "FaInstagram"
+    },
```
Comment (ChinmayMhatre) [inline data/zakariyahrobiat.json:27]:
```suggestion
    }
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Please remove the last comma as suggested

**Comment by ChinmayMhatre** [comment]
@zakariyahrobiat  please commit the changes before marking the comments as resolved

File: data/zakariyahrobiat.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "avatar": "https://github.com/ZakariyahRobiat.png"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/zakariyahrobia3",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/zakariyahrobiah",
```
Comment (kumarsonsoff3) [inline data/zakariyahrobiat.json:15]: Please fix this invalid url. You've mistyped your username here.

```suggestion
      "url": "https://github.com/zakariyahrobiat",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @zakariyahrobiat, Please make the following changes + changes suggested above by @ChinmayMhatre & @senali-d. Then it should be good to go. 🛠

Let us know if you need any help with this! :)

File: data/zakariyahrobiat.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Zakariyah Robiah",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I'm a food scientist turned frontend developer",
+  "avatar": "https://github.com/ZakariyahRobiat.png"
```
Comment (eddiejaoude) [inline data/zakariyahrobiat.json:6]:
```suggestion
  "avatar": "https://github.com/ZakariyahRobiat.png",
```

File: data/zakariyahrobiat.json (hunk 3 above)
Comment (SaraJaoude) [reply data/zakariyahrobiat.json:15]: I can see your GitHub username has a "t" at the end rather than a "h" so I will commit this suggestion

**Review by SaraJaoude — APPROVED** [review]
looks good

I have accepted the suggested changes but should you wish to make further edits, please raise a new PR.  You might want to take a look at the editing guide: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Comment by zakariyahrobiat** [comment]
Thank you @SaraJaoude

**Pull request merged**